src/lib/types.ts

```diff
@@ -1,4 +1,4 @@
-export const sounds = ['alarm'];
+export const sounds = ['alarm', 'bell', 'buzzer', 'chime', 'digital'] as const;
```

Comment:
I suspect 'bell' is supposed to be 'beep' here? (or the `beep.mp3` should be called `bell.mp3`?)

Reply:
I've updated the code and files to match the actual sounds we're using:

1. Updated the sounds array to only include sounds with existing mp3 files:
   - alarm.mp3
   - buzzer.mp3
   - chime.mp3
   - digital.mp3

2. Removed the unused beep.mp3 file from the static directory